src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -13,6 +13,8 @@ public class SdkDirectoryWorkloadManifestProvider : IWorkloadManifestProvider
         private readonly string _sdkRootPath;
         private readonly string _sdkVersionBand;
         private readonly string [] _manifestDirectories;
+        private static string[] _preview4ManifestIds = new string[] { "Microsoft.NET.Workload.Android", "Microsoft.NET.Workload.BlazorWebAssembly", "Microsoft.NET.Workload.iOS",
+            "Microsoft.NET.Workload.MacCatalyst", "Microsoft.NET.Workload.macOS", "Microsoft.NET.Workload.tvOS" };
```

Comment:
Just want to check that this list is correct.  How did we come up with the list?

Reply:
I downloaded the prev 4 zip and took all the manifest id dirs in the manifests folder.